modules/op-release-notes-1-2.adoc

```diff
@@ -26,7 +26,7 @@ In addition to the fixes and stability improvements, the following sections high
 +
 [NOTE]
 ====
-Installations in restricted environments are currently not supported on IBM Power Systems, IBM Z, and LinuxONE.
+Installations in restricted environments are currently not supported on IBM Power Systems, {ibm-z-name}, and LinuxONE.
```

Comment:
Same here, `... not supported on {ibm-power-name}, {ibm-z-name}, and {ibm-linuxone-name}.`